Units/parser-myrddin.r/simple.d/expected.tags

```diff
@@ -0,0 +1,12 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
+!_TAG_FILE_SORTED	1	/0=unsorted, 1=sorted, 2=foldcase/
+!_TAG_PROGRAM_AUTHOR	Universal Ctags Team	//
+!_TAG_PROGRAM_NAME	Universal Ctags	/Derived from Exuberant Ctags/
+!_TAG_PROGRAM_URL	https://ctags.io/	/official site/
+!_TAG_PROGRAM_VERSION	0.0.0	/4d580c2/
```

Comment:
Pseudo tags started from `!_TAG_` can be removed for making expected.tags smaller.
When comparing the output of target ctags and expected.tags, these pseudo tags are ignored.